README.md

```diff
@@ -339,6 +339,7 @@ Additional lists you might find useful:
 - [Cors plugin](https://github.com/snelg/cakephp-cors) - A lightweight plugin for adding CORS headers to specified endpoints.
 - [CrudJsonApi plugin](https://github.com/FriendsOfCake/crud-json-api) - Crud listener for building [JSON API](https://jsonapi.org/) compliant APIs.
 - [FractalTransformerView plugin](https://github.com/andrej-griniuk/cakephp-fractal-transformer-view) - A plugin which allows using [Fractal transformers](https://fractal.thephpleague.com/transformers/) for your API output.
+- :strawberry: [SwaggerBake plugin](https://github.com/cnizzardini/cakephp-swagger-bake) - This plugin automatically builds your Swagger UI (v3.25) from your existing cake models and routes. A redoc option is also available.
```

Comment:
specifics and version numbers get outdated. I would prefer this to be kept agnostic and instead put that detail into plugin readme etc.

Reply:
cake => can be removed
This plugin => can be removed: Builds ...

Reply:
I removed swagger UI version number reference and redundant mention of cake.